src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,42 @@
 package core.basesyntax;
 
+abstract class Machine {
+    public abstract void doWork();
+    public abstract void stopWork();
+}
+class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck start");
```

Comment:
The message 'Truck start' is not very informative. It would be better to indicate what the Truck is doing, e.g., 'Truck is starting its work'.